blocks/search-results/search-results.js

```diff
@@ -132,7 +132,9 @@ async function searchPages(placeholders, term, page) {
     header.appendChild(link);
     res.appendChild(header);
     const para = document.createElement('p');
-    setResultValue(para, line.description, term);
+    if (getLanguage() === 'jp') {
+      setResultValue(para, line.description.split('。')[0], term);
+    } else setResultValue(para, line.description, term);
```

Comment:
Why just `jp` ? Can't the same problem happen in any language with long description?

Reply:
Only `jp` has `。` so there should be another `if` for the other languages that use `.`

Reply:
Also maybe add `ja` as well to be safe, I don't think we have them mixed anywhere but better safe than sorry

Reply:
I mean what if the `line.description.` is long enough to require trimming.